tests/foreman/ui/test_activationkey.py

```diff
@@ -211,378 +158,236 @@ def test_positive_create_activation_key_3(self, env_name):
         repo_id = self.create_sync_custom_repo()
         self.cv_publish_promote(cv_name, env_name, repo_id)
         with Session(self.browser) as session:
             make_activationkey(
                 session,
-                org=self.org_name,
+                org=self.organization.name,
                 name=name,
                 env=env_name,
-                description=gen_string('alpha', 16),
-                content_view=cv_name
+                content_view=cv_name,
             )
             self.assertIsNotNone(self.activationkey.search_key(name))
 
-    @data(*valid_data_list())
-    def test_positive_create_activation_key_5(self, hc_name):
-        """@Test: Create Activation key for all variations of Host Collections
+    def test_positive_create_activation_key_with_hc(self):
+        """@Test: Create Activation key with Host Collection
 
         @Feature: Activation key - Positive Create
 
-        @Steps:
-        1. Create Activation key for all valid Host Collections in [1]
-        using valid Name, Description, Environment, Content View, Usage limit
-
         @Assert: Activation key is created
 
         """
         name = gen_string(str_type='alpha')
-
         # create Host Collection using API
         host_col = entities.HostCollection(
-            organization=self.org_id,
-            name=hc_name
-        ).create_json()
+            organization=self.organization,
+            name=gen_string(str_type='utf8'),
+        ).create()
 
         with Session(self.browser) as session:
             make_activationkey(
                 session,
-                org=self.org_name,
+                org=self.organization.name,
                 name=name,
-                env=ENVIRONMENT
+                env=ENVIRONMENT,
             )
             self.assertIsNotNone(self.activationkey.search_key(name))
             # add Host Collection
-            self.activationkey.add_host_collection(name, host_col['name'])
-            result = self.activationkey.find_element(
-                common_locators['alert.success'])
-            self.assertIsNotNone(result)
+            self.activationkey.add_host_collection(name, host_col.name)
+            self.assertIsNotNone(self.activationkey.find_element(
+                common_locators['alert.success']))
 
             # check added host collection is listed
-            self.activationkey.wait_until_element(
-                tab_locators['ak.host_collections.list']
-            ).click()
+            self.activationkey.click(tab_locators['ak.host_collections.list'])
             strategy, value = tab_locators['ak.host_collections.add.select']
             host_collection = self.activationkey.wait_until_element(
-                (strategy, value % host_col['name']))
+                (strategy, value % host_col.name))
             self.assertIsNotNone(host_collection)
 
-    def test_positive_create_activation_key_6(self):
```

Comment:
That is a default scenario which is definitely covered in scope of basic scenario `test_positive_create_activation_key_with_different_names`